packages/core/addon/mirage/fixtures/reports.js

```diff
@@ -688,5 +688,67 @@ export default [
       ],
       sorts: []
     }
+  },
+  {
+    id: 14,
+    title: 'RequestV2 mmuti-param testing report',
+    createdOn: '2015-04-01 00:00:00',
+    updatedOn: '2015-04-01 00:00:00',
+    authorId: 'navi_user',
+    deliveryRuleIds: [],
+    visualization: {
+      type: 'table',
+      version: 2,
+      metadata: {
+        columnAttributes: {
+          c1: { canAggregateSubtotal: false },
+          c3: { canAggregateSubtotal: false },
+          c2: { canAggregateSubtotal: false }
+        },
+        showTotals: []
+      }
+    },
+    request: {
+      table: 'network',
+      dataSource: 'bardOne',
+      limit: null,
+      requestVersion: '2.0',
+      filters: [
+        {
+          type: 'timeDimension',
+          dataSource: 'bardOne',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          operator: 'bet',
+          values: ['2015-10-02', '2015-10-14']
+        }
+      ],
+      columns: [
+        {
+          cid: 'c1',
+          field: 'network.dateTime',
+          parameters: {
+            grain: 'day'
+          },
+          type: 'timeDimension'
+        },
+        {
+          cid: 'c2',
+          type: 'metric',
+          field: 'revenue',
+          parameters: {
+            currency: 'EUR',
+            as: 'm2'
```

Comment:
`as` is a v1 thing
```suggestion
            currency: 'EUR'
```

Reply:
Do we have any other params we could use to test the multi param capability?